SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -295,6 +295,37 @@ bool USpatialPackageMapClient::SerializeObject(FArchive& Ar, UClass* InClass, UO
 	return true;
 }
 
+const FClassInfo* USpatialPackageMapClient::TryResolveNewDynamicSubobjectAndGetClassInfo(UObject* Object)
+{
+	AActor* Actor = Object->GetTypedOuter<AActor>();
```

Comment:
Ensure Object isn't null before dereferencing

Reply:
Should this be a check?
Or return nullptr and log error?

Reply:
Or actually, I will let Actor be nullptr if Object is nullptr and let one of the other logs handle it.

Reply:
Perfect